### DISCUSSION TITLE
Audiomoth Microphone

### DISCUSSION THREAD

**Discussion by tomwhite101** [mcguirepr89/BirdNET-Pi#751] [Q&A]
Hi All,
I have an audiomoth which I am trying to use as a USB mic.
The pi does not not seem to recognise it as a microphone. I know the pi is working correctly because when I plug in other USB microphones it works.
It is powering the audio moth as it has led flashing.
Any insights on how to make the pi recognise it?

Many thanks,
Tom

**Comment by tomwhite101** [comment]
For info I am using the audiomoth on this basis:
https://github.com/OpenAcousticDevices/Application-Notes/blob/master/Using_the_AudioMoth_USB_Microphone_Firmware_and_Hardware/Using_the_AudioMoth_USB_Microphone_Firmware_and_Hardware.pdf

**Comment by gilpayson** [comment]
I am having the same issue - I have tested the audiomoth usb microphone on a Windoz machine, all ok.  

I am having luck switching it to "default"  have you tried that?  DMESG calls it Product: 250kHz AudioMoth USB Microphone
when I switch it into this mode, but just AUDIOMOTH in the USB/OFF  position
